bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -47,6 +47,7 @@
 			<channel id="moving" typeId="moving"/>
 			<channel id="command" typeId="command"/>
 			<channel id="t4command" typeId="t4command"/>
+			<channel id="courtesy" typeId="courtesy"/>
```

Comment:
You need to add update instructions for this new channel.

Reply:
Ok, done